config/flavors/dev/sc-config.yaml

```diff
@@ -50,3 +119,85 @@ prometheus:
 
 fluentd:
   enabled: false
+
+thanos:
+ query:
+   topologySpreadConstraints: []
+   resources:
+      requests:
+        cpu: 10m
+        memory: 64Mi
+      limits:
+        cpu: 200m
+        memory: 256Mi
+
+ queryFrontend:
+    resources:
+      requests:
+        cpu: 10m
+        memory: 32Mi
+      limits:
+        cpu: 100m
+        memory: 128Mi
+
+ ruler:
+  replicaCount: 1
+  topologySpreadConstraints: []
+  resources:
+      requests:
+        cpu: 10m
+        memory: 64Mi
+      limits:
+        cpu: 200m
+        memory: 128Mi
+
+ receiveDistributor:
+    replicaCount: 1
+    replicationFactor: 1
+    resources:
+      requests:
+        cpu: 20m
+        memory: 64Mi
+      limits:
+        cpu: 200m
+        memory: 256Mi
+
+ receiver:
+   mode: standalone
```

Comment:
Standalone mode disables receive distributor, which isn't supported by the netpols, as they only accept traffic through the distributor, so this needs to be fixed or reverted.